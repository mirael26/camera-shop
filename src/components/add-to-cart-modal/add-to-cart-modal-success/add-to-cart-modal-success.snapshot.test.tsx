import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import AddToCartModalSuccess from './add-to-cart-modal-success';

test('AddToCartModalSuccess component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<AddToCartModalSuccess/>);
  expect(asFragment()).toMatchSnapshot();
});
