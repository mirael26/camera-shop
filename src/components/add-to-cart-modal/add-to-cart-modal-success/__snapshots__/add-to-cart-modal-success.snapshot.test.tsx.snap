// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToCartModalSuccess component renders correctly 1`] = `
<DocumentFragment>
  <p
    class="title title--h4"
  >
    Товар успешно добавлен в корзину
  </p>
  <svg
    aria-hidden="true"
    class="modal__icon"
    height="80"
    width="86"
  >
    <use
      xlink:href="#icon-success"
    />
  </svg>
  <div
    class="modal__buttons"
  >
    <button
      class="btn btn--transparent modal__btn"
    >
      Продолжить покупки
    </button>
    <button
      class="btn btn--purple modal__btn modal__btn--fit-width"
    >
      Перейти в корзину
    </button>
  </div>
  <button
    aria-label="Закрыть попап"
    class="cross-btn"
    type="button"
  >
    <svg
      aria-hidden="true"
      height="10"
      width="10"
    >
      <use
        xlink:href="#icon-close"
      />
    </svg>
  </button>
</DocumentFragment>
`;
